src/__mocks__/octokit/index.js

```diff
@@ -1,34 +1,63 @@
 const createResponse = () => Promise.resolve({
     data: {
-        number: 134,
-        url: 'https://api.github.com/repos/octocat/Hello-World/pulls/134',
-        title: "Amazing new feature",
+        number: 1,
+        url: 'https://api.github.com/repos/sbsrnt/Hello-World/pulls/1',
```

Comment:
I couldn't find opened/closed PR that had `labels` in `octocat`'s so I [created my own test PRs](https://github.com/sbsrnt/Hello-World/pulls). 

I would say it would be better if this repo had own test PRs for better integrity.

Reply:
PRs for users to be able to view the final result of the settings? :thinking:

Reply:
Naaa, just for mocking possibilities. It feels like relying on `octocat`'s repo you are limiting yourself with new feature releases.

In case if someone decides to close said PRs, mocks will be failing (doubt that may happen but you know how it is 😄)

Reply:
Hm, yes, I understand your point of view, however I'm not sure how often a case like the one you mentioned happens.

But anyway I like the idea and will follow it. Soon I will make some changes following what you commented. Thanks for the sugestion